app/models/host.rb

```diff
@@ -1859,6 +1859,10 @@ def read_only_storages
     storages.where(:host_storages => {:read_only => true})
   end
 
+  def available_storages
+    writable_storages.reject(&:maintenance)
```

Comment:
@agrare @Fryguy Is `available_storages` the right name?  read_only storage is still available (for reading) just not for our intended purpose which is provisioning (writing).

Should this be something like `provisionable_storage`?